javaparser-core/src/main/java/com/github/javaparser/CommentsInserter.java

```diff
@@ -192,11 +203,14 @@ private boolean attributeLineCommentToNodeOrChild(Node node, LineComment lineCom
     }
 
     private boolean thereAreLinesBetween(Node a, Node b) {
+        if (!a.getRange().isPresent() || !b.getRange().isPresent()) {
+            return true;
+        }
         if (!PositionUtils.areInOrder(a, b)) {
             return thereAreLinesBetween(b, a);
         }
-        int endOfA = a.getEnd().line;
-        return b.getBegin().line > (endOfA + 1);
+        int endOfA = a.getRange().get().end.line;
```

Comment:
Maybe this is a little bit too long to write. Should we create & use a `getEndLine` method?